### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abhishekshahj**
Updated this md as now that blog is not live or been deleted by author.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,6 @@ This module enables you to easily get the protocol and host of your Next.js app,
 
 ## Usage
 
-> I'm honored that an excellent blog post has been published about this package. For detailed usage, please see: https://codeconqueror.com/blog/get-the-current-url-in-next-js
-
-```js
```
Comment (jakeburden) [inline README.md:11]: Looks like this line was accidentally deleted.

**Comment by jakeburden** [comment]
Should we use an archived link of the blog?
https://web.archive.org/web/20211110110818/https://codeconqueror.com/blog/get-the-current-url-in-next-js/

**Comment by abhishekshahj** [comment]
Yes, we can use web archived link for this blog.
https://web.archive.org/web/20211110110818/https://codeconqueror.com/blog/get-the-current-url-in-next-js/

**Comment by jakeburden** [comment]
Closing in favor of: https://github.com/jakeburden/next-absolute-url/pull/28

**Pull request closed without merging**